### PR TITLE
widget fixes and cleanup

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixes widget
- js files cleanup after merged pr https://github.com/divio/django-filer/commit/5620532fbf1f20a955ae41bc67d79673e77d1619?diff=split#diff-7f831ee1b65c09f6e40970d44d0e2cdc
- no css and js include in widget template
- general style fixes

### CODE REVIEW

**Comment by vxsx** [comment]
LGTM, however, to avoid variable renaming in the future we should probably make a point in the docs about FE guidelines http://django-filer.readthedocs.org/en/latest/development.html#working-on-front-end
(regarding $var for example https://aldryn-boilerplate-bootstrap3.readthedocs.org/en/latest/guidelines/javascript.html#naming)

**Comment by vxsx** [comment]
not necessarily in this PR

File: filer/static/filer/js/addons/widget.js [hunk 1]
```diff
@@ -3,27 +3,26 @@
...
-        var $description = $container.find('.description_txt');
-        var $addImageButton = $container.find('.lookup');
-        var $dropzoneMessage = $container.siblings('.dz-message');
         var clearer = $(this);
+        var container = clearer.closest('.filerFile');
+        var input = container.find(':input');
+        var thumbnail = container.find('.thumbnail_img');
+        var description = container.find('.description_text');
+        var addImageButton = container.find('.lookup');
+        var $dropzoneMessage = container.siblings('.dz-message');
```
Comment (stefanfoulis) [inline filer/static/filer/js/addons/widget.js]: missed one :-)

**Comment by stefanfoulis** [comment]
:+1:

**Pull request merged**